requirements/ci/py37.yml

```diff
@@ -34,9 +34,10 @@ dependencies:
 # Test dependencies.
   - asv
   - filelock
```

Comment:
Just curious - but where did black go, its in the test dependencies in the v3.0.x release branch?

Reply:
The `black` dependency was dropped from `main` by https://github.com/SciTools/iris/pull/4181

Reply:
Ah - missed that.  Thanks.